### PR TITLE
Sambamba tools split according to functionality

### PR DESCRIPTION
**Pull request by thaze**
@bgruening and @TorHou : Here are the wrappers for Sambamba view, flagstat, slice and sort. Please review.

### CODE REVIEW

**Comment by bgruening** [comment]
Can you please add a `.shed.yml` file, like this one: https://github.com/galaxyproject/tools-iuc/blob/master/tools/gemini/.shed.yml

This will trigger automatic test testing for this repository.

**Comment by thaze** [comment]
@TorHou  and @bgruening: I've added '.shed.yml' and the tools for depth, merge and markdup. Please review. Thanks.

File: tools/sambamba/Sambamba_merge.xml [hunk 1]
```diff
@@ -0,0 +1,52 @@
+<tool id="sambamba_merge" name="Merge" version="@WRAPPER_VERSION@">
+    <description>Merge several BAM files into one!</description>
+    <macros>
+        <import>Sambamba_macros.xml</import>
+    </macros>
+    <expand macro="requirements" />
+    <expand macro="stdio" />
+    <command>
+<![CDATA[
+        ln -s ${input_file1} infile1.bam &&
```
Comment (yhoogstrate) [inline tools/sambamba/Sambamba_merge.xml]: missing quotes

**Comment by bgruening** [comment]
The file needs to be called `.shed.yml` with a leading dot :)

**Comment by thaze** [comment]
@bgruening : The automated tests on travis failed, looks like a tool dependency issue. I need guidance on how to resolve it :)

File: tools/sambamba/Sambamba_macros.xml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+            <exit_code range=":-1" />
+            <!-- In case the return code has not been set propery check stderr too -->
+            <regex match="Error:" />
+            <regex match="Exception:" />
+        </stdio>
+    </xml>
+
+    <token name="@EXECUTABLE">
+    <![CDATA[
+        $__tool_directory__/sambamba
```
Comment (bgruening) [inline tools/sambamba/Sambamba_macros.xml]: `$__tool_directory__/` needs to go away

File: tools/sambamba/Sambamba_macros.xml [hunk 3]
```diff
@@ -0,0 +1,59 @@
+<macros>
+
+    <xml name="requirements">
+        <requirements>
+            <requirement type="package" version="0.6.1">Sambamba</requirement>
```
Comment (bgruening) [inline tools/sambamba/Sambamba_macros.xml]: lower `s`

**Comment by yhoogstrate** [comment]
I have added some changes (https://github.com/thaze/galaxytools/pull/3)

**Pull request merged**